lib/plugins/aws/deploy/index.js

```diff
@@ -32,8 +33,9 @@ class AwsDeploy {
     Object.assign(
       this,
       extendedValidate,
-      createStack,
       setBucketName,
+      shouldBeDeployed,
```

Comment:
Good one. TBH I'm also slightly embarrassed with the name.

I was literally sitting there yesterday trying to come up with an appropriate name.

But you know:

> There are only two hard things in Computer Science, ...